tests/e2e/e2e_test.go

```diff
@@ -627,7 +627,7 @@ var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
 			ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
 			_, err := cli.Health(ctx)
 			cancel()
-			gomega.Ω(err.Error()).Should(gomega.ContainSubstring("not bootstrapped"))
+			gomega.Ω(err.Error()).Should(gomega.ContainSubstring(server.ErrNetworkNotRunning.Error()))
```

Comment:
It wasn't accurate -- it was saying "not bootstrapped" but really the error is that the network isn't running at all

Reply:
Could we keep the original naming? This will propagate into CLI and represents and API change, would like to include it in a bigger refactor related to API changes with appropiate semver change.